stubs/redis/redis/connection.pyi

```diff
@@ -149,65 +164,59 @@ class SSLConnection(Connection):
     ) -> None: ...
 
 class UnixDomainSocketConnection(Connection):
-    description_format: Any
-    pid: Any
-    path: Any
-    db: Any
-    password: Any
-    socket_timeout: Any
-    retry_on_timeout: Any
-    encoding: Any
-    encoding_errors: Any
-    decode_responses: Any
-    retry: Retry
+    path: str
     def __init__(
         self,
-        path=...,
+        path: str = ...,
         db: int = ...,
-        username=...,
-        password=...,
-        socket_timeout=...,
+        username: str | None = ...,
+        password: str | None = ...,
+        socket_timeout: float | None = ...,
         encoding: str = ...,
         encoding_errors: str = ...,
         decode_responses: bool = ...,
         retry_on_timeout: bool = ...,
-        retry_on_error=...,
-        parser_class=...,
+        retry_on_error: list[type[Exception]] = ...,
+        parser_class: type[BaseParser] = ...,
         socket_read_size: int = ...,
         health_check_interval: int = ...,
-        client_name=...,
+        client_name: str | None = ...,
         retry: Retry | None = ...,
         redis_connect_func: _ConnectFunc | None = ...,
     ) -> None: ...
-    def repr_pieces(self) -> list[tuple[str, str]]: ...
 
+# TODO: make generic on `connection_class`
 class ConnectionPool:
+    connection_class: type[Connection]
+    connection_kwargs: Any
+    max_connections: int
+    pid: int
     @classmethod
     def from_url(cls: type[Self], url: str, *, db: int = ..., decode_components: bool = ..., **kwargs) -> Self: ...
-    connection_class: Any
-    connection_kwargs: Any
-    max_connections: Any
-    def __init__(self, connection_class=..., max_connections=..., **connection_kwargs) -> None: ...
-    pid: Any
-    def reset(self): ...
-    def get_connection(self, command_name, *keys, **options: _ConnectionPoolOptions): ...
-    def make_connection(self): ...
-    def release(self, connection): ...
-    def disconnect(self, inuse_connections: bool = ...): ...
+    def __init__(
+        self, connection_class: type[Connection] = ..., max_connections: int | None = ..., **connection_kwargs
+    ) -> None: ...
+    def reset(self) -> None: ...
+    def get_connection(self, command_name: Any, *keys, **options: _ConnectionPoolOptions) -> Connection: ...
```

Comment:
`command_name` is an entirely unused parameter, if I'm reading the source code correctly, so `object` is better than `Any` here

```suggestion
    def get_connection(self, command_name: object, *keys, **options: _ConnectionPoolOptions) -> Connection: ...
```